source/user-manual/capabilities/file-integrity/fim-configuration.rst

```diff
@@ -5,33 +5,33 @@
 Configuration
 =============
 
-#. `Basic usage`_
-#. `Configuring scheduled scans`_
+After the installation, the Wazuh manager and the Wazuh agent come with the :ref:`default syscheck configuration <reference_ossec_syscheck_default_configuration>`, which should be reviewed and adjusted to the particular environment. This section presents different configuration examples, helping to learn how to use syscheck in different use cases:
```

Comment:
What if we change `the Wazuh agent come with the ` for `the Wazuh agent have defined a `? Is this `come with` commonly used in this kind of documents?

Reply:
Done